chain/chain/src/resharding/manager.rs

```diff
@@ -216,7 +216,7 @@ impl ReshardingManager {
                 "Creating child memtrie by retaining nodes in parent memtrie..."
             );
             let mut mem_tries = mem_tries.write().unwrap();
-            let mut trie_recorder = TrieRecorder::new();
+            let mut trie_recorder = TrieRecorder::new(None);
```

Comment:
it would be interesting to see what's the proof size during resharding in forknet
not for this pr, we can probably get it from some metrics or logs